### DISCUSSION TITLE
easy install games for PSX DVR and BBNavigator

### DISCUSSION THREAD

**Discussion by Tiagofanhais** [israpps/HDL-Batch-installer#23] [Ideas]
hello  can you make this app to help install games  into BBnavigator because for HDD-OSD i can but for BB not and I don't know where i'm doing wrong
and your app is awesome very easy to use ´
thank you for develop this tool
Obrigado, Gracias

**Comment by israpps** [comment]
HI!

There is a big problem with PS2BBN and PSX DVR.

Both softwares can´t load games from the data partition directly.

I´m not an expert on PS2BBN usage and I Don´t own a PSX (unfortunately)


However, (AS FAR AS I KNOW)

The game must be installed with the hidden prefix (aka: `__.`)
OPL Launcher (and PSX XMB Files) must be located on a partition named the same way, but with the parent partition Prefix (aka: `PP.`)

Anyway.

In a moment i´ll lend you the URL of the official PS2 Scene discord.

There, you can find several users and devs (Even krHACKen) who own a PSX, they will be capable of providing exact and accurate information!
